### PR TITLE
bootstrap from prefect collection template.

### PR DESCRIPTION
**Pull request by tardunge**
- generate from the boilerplate collection template provided by prefect.

### CODE REVIEW

**Comment by limx0** [comment]
Thanks @tardunge! 

Is this ready to merge?

**Comment by tardunge** [comment]
> Thanks @tardunge!
> 
> Is this ready to merge?

This is just `Ready to Review`. Need some modifications to be done for placeholder values. Can you help with them a bit?
Need some logo and docs hosting service.

File: .cruft.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "template": "https://github.com/PrefectHQ/prefect-collection-template",
+  "commit": "e11a3be195f24f60ed3f564dfccb40170ee7c3fa",
+  "checkout": null,
+  "context": {
+    "cookiecutter": {
+      "full_name": "Arthur Dent",
+      "email": "arthur.dent@example.com",
```
Comment (tardunge) [inline .cruft.json]: update maintainer email id.

File: .cruft.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "github_organization": "arthur_dent",
```
Comment (tardunge) [inline .cruft.json]: update maintainer org id

File: .cruft.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      "collection_name": "prefect-lakefs",
+      "collection_slug": "prefect_lakefs",
+      "collection_short_description": "Prefect Collection Template contains all the boilerplate that you need to create a Prefect collection.",
```
Comment (tardunge) [inline .cruft.json:12]: change short desc to something meaniful.
keeps short desc consistent accross cruft updates.

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      - name: Download build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: built-package
+          path: "./dist"
+
+      - name: Publish distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (tardunge) [inline .github/workflows/release.yml:51]: Need PYPI token to publish.

**Comment by tardunge** [comment]
> > Thanks @tardunge!
> > Is this ready to merge?
> 
> This is just `Ready to Review`. Need some modifications to be done for placeholder values. Can you help with them a bit? Need some logo and docs hosting service.

Just marked some places where i might need your help.

**Pull request merged**